src/endpoints/users-endpoint.ts

```diff
@@ -0,0 +1,26 @@
+import { Endpoint, Request, Response } from 'death-adder';
+
+class UsersEndpoint extends Endpoint {
+  // get all users
+  async get(request: Request, response: Response) {
+    const users = await request.attachments.db.query(
```

Comment:
So, here and below I can see that "db is not defined on Object". I see that there aren't any `package.json` changes either to bump up to the new beta for death adder, chance you didn't commit them?